hoomd/md/EvaluatorPairGB.h

```diff
@@ -74,7 +57,47 @@ struct gb_shape_params
 class EvaluatorPairGB
     {
     public:
-        typedef pair_gb_params param_type;
+        struct param_type
+            {
+            Scalar epsilon;     //! The energy scale.
+            Scalar lperp;       //! The semiaxis length perpendicular to the particle orientation.
+            Scalar lpar;        //! The semiaxis length parallel to the particle orientation.
+
+            #ifdef ENABLE_HIP
+            //! Set CUDA memory hints
+            void set_memory_hint() const
+                {
+                // default implementation does nothing
+                }
+            #endif
+
+            #ifndef __HIPCC__
+            param_type() {epsilon = 0; lperp = 0; lpar = 0;}
+
+            param_type(pybind11::dict v)
+                {
+                auto epsilon(v["epsilon"].cast<Scalar>());
+                auto lperp(v["lperp"].cast<Scalar>());
+                auto lpar(v["lpar"].cast<Scalar>());
```

Comment:
This will not set `epsilon`, `lperp` or `lpar`. `epsilon = v["epsilon"].cast<Scalar>();`